src/pages/kayoung/Main/KayoungMain.js

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import './KayoungMain.scss';
+import Nav from '../../../components/Nav/Nav';
+import Comment from './Comment';
+
+class KayoungMain extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      replyBox: [],
+      replyContent: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          replyBox: data,
+        });
+      });
+  }
+
+  storeComment = e => {
+    this.setState({ replyContent: e.target.value });
+    if (e.key === 'Enter') {
+      this.addComment(e);
+    }
+  };
+
+  addComment = e => {
+    e.preventDefault();
+
+    this.setState({
+      replyBox: this.state.replyBox.concat(this.state.replyContent),
+      replyContent: '',
+    });
```

Comment:
```suggestion
    e.preventDefault();
    const {replyBox, replyContent} = this.state;
    this.setState({
      replyBox: replyBox.concat(replyContent),
      replyContent: '',
    });
```
구조분해 해서 사용해도 깔끔해보입니다.